recipes/perfetto/all/CMakeLists.txt

```diff
@@ -4,12 +4,10 @@ project(perfetto CXX)
 include(conanbuildinfo.cmake)
 conan_basic_setup()
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-
 set(PUBLIC_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder/sdk/perfetto.h)
 
 add_library(perfetto source_subfolder/sdk/perfetto.cc)
+target_compile_features(perfetto PRIVATE cxx_std_11 cxx_constexpr)
```

Comment:
Could you also bump min CMake version to 3.8 please (for cxx_std feature)?

Reply:
Done